src/components/user/avatar/UserAvatar.tsx

```diff
@@ -10,19 +9,19 @@ import User from '../../../types/User';
 import Utils from '../../../utils/Utils';
 import computeStyleSheet from './UserAvatarStyle';
 
-interface State {
-}
+interface State {}
 
 export interface Props extends BaseProps {
   user?: User;
   accessoryIcon?: string;
-  selected?: boolean;
   size?: number;
+  isSelected?: boolean;
 }
 
 export default class UserAvatar extends React.Component<Props, State> {
+  public static defaultProps: { size: number; isSelected: false };
   private centralServerProvider: CentralServerProvider;
-
+  private commonColors = Utils.getCurrentCommonColor();
```

Comment:
Should be used in render() methods because the theme can change dynamically (dark/light)